es-ds-docs/pages/molecules/pagination.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div>
+        <h1>Pagination</h1>
+        <p class="pb-200">
+            Extended from <nuxt-link
+                href="https://v3.primevue.org/paginator/"
+                target="_blank">
+                PrimeVue paginator
+            </nuxt-link>
+        </p>
+
+        <div class="mb-500">
+            <ul
+                id="paginated-list">
+                <li
+                    v-for="item in items.slice(first, first + perPage)"
+                    :key="item">
+                    {{ item }}
+                </li>
+            </ul>
+            <es-pagination
+                v-model:first="first"
```

Comment:
So currently people just pass a v-model with the current page, rather than the "first" which corresponds to the record number that's at the start of the current page. How would you feel about matching the existing API we have for pagination? Our best practices are still being developed, but the [current guidance](https://energysage.atlassian.net/wiki/spaces/CED/pages/1009025072/PrimeVue+and+Nuxt+3+implementation+guide#General-guidance) is "to provide a wrapper component of the same name in ESDS 3.0 that matches the props API of the original component... The only exception to this is if creating an ESDS-2.0-style wrapper component around a PrimeVue component would be so different from how the PrimeVue component works as to take a long time or be overly complex."

I'm happy to strategize with you on this- I think it would just be a matter of defining a model on `es-pagination` which would then be translated to and from PrimeVue's format based on the number of items per page.

Reply:
That's fair! I was originally thinking it should be a really easy switch downstream, but it's also not nothing and would require the arithmetic to change for how list items are displayed.  Implemented by defining a model for `currentPage` from the parent and setting up reactivity as you suggested